bindings/pydeck/Makefile

```diff
@@ -18,11 +18,11 @@ init:
 	yarn bootstrap; \
 	$(BIN)pip install -r requirements/requirements.txt; \
 	$(BIN)pip install -r requirements/requirements-dev.txt; \
-	$(BIN)pip install -e .;
+	python setup.py develop
 
 fast-build:
 	cd ../../modules/jupyter-widget; webpack; cd -; \
-	pip install -e .
+	python setup.py develop
```

Comment:
nit: I can't find the original blog post right now but calling `python setup.py` directly is deprecated